apps/pagetour-sdk/src/common/configstore.ts

```diff
@@ -67,13 +67,34 @@ class ConfigStore {
     },
     zIndex: "100000",
     textAreaCharacterLength: 500,
-    enableBeacon: false
+    enableBeacon: false,
+    feedback : {
+      pagetourFeedbackOptions: {
+        enabled: false,
+        type:'5-star',
+        heading: "Feedback : PageTour",
+        description: "Was this tour helpful?",
+        privacyDescription: "Privacy statement",
+        privacyURL: 'https://privacy.microsoft.com/en-US/data-privacy-notice'
+      },
+      announcementFeedbackOptions: {
+        enabled: false,
+        type: 'like-dislike',
+        heading: 'Was this announcement helpful?',
+        privacyDescription: "Privacy statement",
+        privacyURL: 'https://privacy.microsoft.com/en-US/data-privacy-notice',
```

Comment:
I think for the default one we should not put privacy link. User who wants the privacy link will add it

Reply:
Removed privacy link for default options. it will not be added if user has not added privacy link themselves.